buildozer/targets/osx.py

```diff
@@ -44,16 +45,19 @@ def download_kivy(self, cwd):
         else:
             if not exists(join(cwd, 'Kivy.dmg')):
                 self.logger.info('Downloading kivy...')
-                status_code = check_output((
-                    'curl', '-L', '--write-out', '%{http_code}',
-                    '-o', 'Kivy.dmg',
-                    f'https://kivy.org/downloads/{current_kivy_vers}/Kivy.dmg'),
-                    cwd=cwd)
-
-                if status_code == "404":
+                try:
+                    buildops.download(
+                        f'https://kivy.org/downloads/'
+                        f'{current_kivy_vers}/Kivy.dmg',
```

Comment:
Minor: Can we improve the readability by avoiding the string slicing?

Reply:
The full string went over the 79/80 character line limit, so I split it in two.

Would putting them in parenthesis and adding them together help?

I will go back to breaking the line length rules for now, so this minor issue isn't blocking us.